### ISSUE TITLE
Bedtools install on Mac

### ISSUE THREAD

**Issue by luisymbio** [arq5x/bedtools2#1077]
Hi all, 

I'm trying to install a package which has a dependencies based on bedtools, the hardware is a Macbook pro. I read about people that got the installation through Conda channels, but conda install or mamba install give me the following output:
`bedtools   does not exist (perhaps a typo or a missing channel).
`

Do you have some solutions or know where I can find a way to install it?

**Comment by biogabriel7** [comment]
Hi! first, make sure you are running conda on osx64 and not osx-arm64. Then, check your python version, I don't want to say any lies but I'm pretty sure that conda-forge / bioconda only works with Python 3.7 - 3.10 (you can always check it on their wiki). 

This is the most typical mistake when dealing with this type of message. Hope you solved it!

**Comment by luigallucci** [comment]
Yes, I was trying to install it on arm processor. There is a way to install Bedtools with this difference?

**Comment by biogabriel7** [comment]
Uninstall everything and go to:
https://conda-forge.org

Install directly the version x86_64 (You have to have downloaded before software meant for Intel proc and have Rosetta on your Mac) you can check on internet how to verify if you have it.

Then, use bash to install the x86_64 version on the terminal. Reset the terminal and `conda info`, you should see  
platform : osx-64

Then, create an env with a version of Python compatible with Bioconda (3.7 - 3.10) and you should be able to download everything!

(While downloading Conda from Conda-Forge it will make these channels per default so no need to config them before install any package)
https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/osx-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
 
This will also add per default mamba, which I strongly recommend to create env / download packs. (it has the same command structure than conda but way faster)

Hope it works!
